P1-HashSet.py

```diff
@@ -0,0 +1,28 @@
+# Leetcode Problem No 705
+# Time Complexity : O(1)
+# Space Complexity : O(1)
+# Runtime: 107ms
+
+class MyHashSet:
+
+    def __init__(self):
+        self.myHashSet = set()
```

Comment:
Do not use inbuilt set to design a hashset. Its nota good thing. Try it with the solution discussed in class.

Reply:
Sure. Let me do it again.